app/views/topup/homepage.html

```diff
@@ -0,0 +1,656 @@
+    <div class="content">
+      <label for="site-search-text">Search</label>
+      <input type="search" name="q" id="site-search-text" title="Search" class="js-search-focus">
+      <input class="submit" type="submit" value="Search">
+    </div>
+  </form>
+
+        </div>
+
+      </div>
+    </header>
+
+      <div id="user-satisfaction-survey-container"></div>
+
+
+    <div id="global-header-bar"></div>
+
+
+
+  <div class="wrapper" id="wrapper">
+
+
+
+<div class="gem-c-breadcrumbs govuk-breadcrumbs " data-module="track-click">
+  <ol class="govuk-breadcrumbs__list">
+      <li class="govuk-breadcrumbs__list-item" aria-current="false">
+          <a data-track-category="homeLinkClicked" data-track-action="homeBreadcrumb" data-track-label="" data-track-options="{}" class="govuk-breadcrumbs__link" aria-current="false" href="/">Home</a>
+      </li>
+      <li class="govuk-breadcrumbs__list-item" aria-current="false">
+          <a data-track-category="breadcrumbClicked" data-track-action="2" data-track-label="/government/organisations" data-track-options='{"dimension28":"3","dimension29":"Organisations"}' class="govuk-breadcrumbs__link" aria-current="false" href="/government/organisations">Organisations</a>
+      </li>
+      <li class="govuk-breadcrumbs__list-item" aria-current="page">
+          <a data-track-category="breadcrumbClicked" data-track-action="3" data-track-label="#content" data-track-options='{"dimension28":"3","dimension29":"Dounreay "}' class="govuk-breadcrumbs__link" aria-current="page" href="#content">UK National Screening Committee</a>
+      </li>
+  </ol>
+</div>
+
+
+
+
+    <main id="content" role="main" class="content ">
+
+
+
+  <h1 class="visually-hidden">UK National Screening Committee</h1>
+
+  <div class="grid-row">
+    <div class="column-two-thirds organisation__margin-bottom">
+      <!-- To make the links sit in the centre column use  <div class="column-one-third organisation__margin-bottom"> -->
+      <div class="gem-c-organisation-logo ">
+    <div class="gem-c-organisation-logo__container gem-c-organisation-logo__crest  "> <img src="/public/images/nsc-logo.jpg" height="39px" width="28px" style="margin-top: -40px"><br>
+      <span class="gem-c-organisation-logo__name">UK National <br>Screening Committee</span>
+    </div>
+</div>
+
+
+    </div>
+
+      <div class="column-one-third organisation__margin-bottom">
+
+          <ol class="app-c-topic-list app-c-topic-list--small brand--department-for-business-innovation-skills ">
+            <li class="app-c-topic-list__item">
+              <a class="app-c-topic-list__link app-c-topic-list__link--no-underline brand__color" href="#role">What the UK National Screening Committee does</a>
+            </li>
+      <li class="app-c-topic-list__item">
+        <a class="app-c-topic-list__link app-c-topic-list__link--no-underline brand__color" href="https://www.gov.uk/guidance/nhs-population-screening-explained">Screening explained</a>
```

Comment:
We had assessment feedback on the homepage which said 'Link headings do not match page headings' - should this be 'NHS Population Screening Explained' ?

Reply:
Aw but I really want to change them! It feels like from the perspective of an 'infrequent visitor' to the site, the links don't make much sense as they currently are. Anyone who is used to the site 'admin' or 'policy stakeholder' doesn't have a problem. I guess we were much more exposed to infrequent visitors this time round and it was really obvious that the links didn't mean much without context. I also get the importance of the assessors comments but also, that they don't have the project context. I wonder if it's gds content design policy to have same link headings or if there is some leeway? I think i will ask Iain first and see what he says (pretty sure he advocated making them less jargon-y!)